test/README.md

```diff
@@ -146,4 +146,4 @@ pytest -v test/test_checkers.py -k python
 
 ## Known issues
 
-If you're using Windows we **strongly** recommend also `pdftotext`. We experienced problems running tests without this. The best approach to do this is through [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html) (click [here](https://anaconda.org/conda-forge/pdftotext) to find out how to install this package with conda).
\ No newline at end of file
+If you're using Windows we **strongly** recommend also `pdftotext`. We experienced problems running tests without this. The best approach to do this is through [conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/windows.html) (click [here](https://anaconda.org/conda-forge/pdftotext) to find out how to install this package with conda).
```

Comment:
@terriko isn't suggested that we install `pdftotext` irrelevant of the OS, but recommended to install it via conda for windows?

Reply:
I think @anthonyharrison 's changes have made it so that it will fail gracefully if pdftotext is not installed regardless of platform, so we don't need this message any more.  It's out of scope for this pull request anyhow and I'm not sure why it's showing up.  Reversion from an earlier version maybe?